android-vjestina-homework-movie-app-data-mock/app/build.gradle

```diff
@@ -54,15 +54,17 @@ dependencies {
     implementation 'com.google.android.material:material:1.6.1'
     implementation "androidx.compose.ui:ui:1.3.0-beta03"
     implementation "androidx.compose.material:material:1.3.0-beta03"
+    implementation "androidx.compose.material3:material3-window-size-class:1.0.0-rc01"
+    implementation "androidx.compose.material3:material3:1.0.0-rc01"
```

Comment:
I've asked your colleague Matej Salković about this, because I've used Compose Material 3 instead of Material Compose. He consulted with other colleagues and said it's ok to use Material 3 and that I only need to create UserScoreProgressBar with canvas and not use template from Material 3